src/session.ts

```diff
@@ -477,7 +481,7 @@ export class SessionManager implements Middleware {
             new PowerShellProcess(
                 this.PowerShellExeDetails.exePath,
                 this.bundledModulesPath,
-                "PowerShell Integrated Console",
+                "PowerShell Extension",
```

Comment:
Explicitly no "Terminal" addendum because this is what's displayed in the list of terminals.